user-guide/settings/privacy.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Privacy
+slug: privacy-submenu
+tags: privacy, delete account, unsubscribe, email list
+publish: true
+position: 40
+---
+
+# Privacy submenu
+
+The __Privacy__ submenu provide options to opt-out of receiving emails from Progress and its partners and the option to delete your Fiddler Everywhere associated account.
+
+There are two options in the submenu:
```

Comment:
There are 3 items below this statement. Consider using the word "several" instead of "two" to avoid ambiguity.